uctMaths/uctMaths/background_tasks.py

```diff
@@ -1,3 +1,4 @@
+from PyPDF2 import PdfFileMerger
```

Comment:
Is this a new library. If so we should add it and all of it's dependencies to req.txt